# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/Worker.cpp
# Commit: f3204b9ec0dc
# Full Hash: f3204b9ec0dc99aad004ce80499052fe5dcd5494
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2023-01-19 04:46:07
# Regressor Bug: 1247687
# File Overlap Count: 1
# Description:
#   Bug 1247687 - Implement requestCredentials for Module Workers; r=jonco
#   
#   This weaves credentials through from the private to script creation.
#   
#   Depends on D155567
# ==============================================================================

diff -r 7327e6438343 -r f3204b9ec0dc dom/workers/Worker.cpp
--- a/dom/workers/Worker.cpp	Wed Jan 18 13:46:31 2023 +0000
+++ b/dom/workers/Worker.cpp	Wed Jan 18 13:46:31 2023 +0000
@@ -42,8 +42,8 @@
 
   RefPtr<WorkerPrivate> workerPrivate = WorkerPrivate::Constructor(
       cx, aScriptURL, false /* aIsChromeWorker */, WorkerKindDedicated,
-      aOptions.mType, aOptions.mName, VoidCString(), nullptr /*aLoadInfo */,
-      aRv);
+      aOptions.mCredentials, aOptions.mType, aOptions.mName, VoidCString(),
+      nullptr /*aLoadInfo */, aRv);
   if (NS_WARN_IF(aRv.Failed())) {
     return nullptr;
   }